tests/acceptance/features/apiSpacesShares/shareSubItemOfSpace.feature

```diff
@@ -66,13 +83,35 @@ Feature: Share a file or folder that is inside a space
       | folder   | viewer    | 404        | No share permission |
 
 
-  Scenario Outline: An user participant of the project space can see the created resources share
+  Scenario Outline: A user participant of the project space can see the created resources share
     Given user "Alice" has shared a space "share sub-item" to user "Brian" with role "<spaceRole>"
-    When user "Alice" shares the following entity "file.txt" inside of space "share sub-item" with user "Bob" with role "editor"
+    When user "Alice" creates a share inside of space "share sub-item" with settings:
+      | path      | file.txt |
+      | shareWith | Bob      |
+      | role      | editor   |
     Then for user "Alice" the space "share sub-item" should contain the last created share of the file "file.txt"
     And for user "Brian" the space "share sub-item" should contain the last created share of the file "file.txt"
     Examples:
       | spaceRole |
       | editor    |
       | viewer    |
       | manager   |
+
+
+  Scenario: A user shares the folder to the group
+    Given group "sales" has been created
+    And the administrator has added a user "Brian" to the group "sales" using GraphApi
+    When user "Alice" creates a share inside of space "share sub-item" with settings:
+      | path       | folder                   |
+      | shareWith  | sales                    |
+      | shareType  | 1                        |
+      | role       | viewer                   |
+      | expireDate | 2042-01-01T23:59:59+0100 |
+    Then the HTTP status code should be "200"
+    And the OCS status code should be "200"
+    And the OCS status message should be "OK"
+    When user "Brian" accepts share "/folder" offered by user "Alice" using the sharing API
+    Then as "Brian" folder "Shares/folder" should exist
+    When user "Brian" gets share "Shares/folder" shared with him using the sharing API
+    Then the fields of the last response to user "Brian" should include
+      | expiration | 2042-01-01T23:59:59+0100 |
```

Comment:
What about combining both steps into one, to make it read easier and avoid When-Then-When-Then series?
```suggestion
    Then the information about the share "Shares/folder" for user "Brian" should include
      | expiration | 2042-01-01T23:59:59+0100 |
```